terraform/20-app/lambda.ingestion.tf

```diff
@@ -34,7 +34,7 @@ module "lambda_ingestion" {
   environment_variables = {
     INGESTION_BUCKET_NAME              = module.s3_ingest.s3_bucket_id
     POSTGRES_DB                        = local.rds.app.primary.db_name
-    POSTGRES_HOST                      = module.rds_proxy.proxy_endpoint
```

Comment:
Pointed straight at the primary for now. Once aurora is in place and has been promoted, I'll point the lambda at that instead